src/displayBackend/BuffersStorage.hpp

```diff
@@ -60,10 +60,12 @@ class BuffersStorage
 	 * @param width          width in pixels
 	 * @param height         height in pixels
 	 * @param bpp            bits per pixel
+	 * @param offset         offset of the data in the buffer
 	 */
 	void createDisplayBuffer(uint64_t dbCookie, bool beAllocRefs,
 							 grant_ref_t startDirectory, uint32_t size,
-							 uint32_t width, uint32_t height, uint32_t bpp);
+							 uint32_t width, uint32_t height,
```

Comment:
I would put offset before size in parameter list. It is more logical: offset, size are buffer parameters.

Reply:
done